packages/core/test/Cardano/util/computeImplicitCoin.test.ts

```diff
@@ -7,14 +7,18 @@ describe('Cardano.util.computeImplicitCoin', () => {
   let dRepPublicKey: Crypto.Ed25519PublicKeyHex;
   let dRepKeyHash: Crypto.Ed25519KeyHashHex;
 
+  beforeAll(async () => {
+    await Crypto.ready();
+  });
```

Comment:
Just FIY: this can be simplified as you just need to return a promise
```suggestion
  beforeAll(() => Crypto.ready());
```